pkg/apis/apis.go

```diff
@@ -19,13 +19,15 @@ package apis
 import (
 	"k8s.io/apimachinery/pkg/runtime"
 
-	"sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+	clusterv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
could we call this `capiv1beta1` to keep it consistent with the other usages

Reply:
ok